routers/repo/commit.go

```diff
@@ -246,6 +248,23 @@ func Diff(ctx *context.Context) {
 	ctx.Data["Parents"] = parents
 	ctx.Data["DiffNotAvailable"] = diff.NumFiles() == 0
 	ctx.Data["SourcePath"] = setting.AppSubURL + "/" + path.Join(userName, repoName, "src", "commit", commitID)
+
+	notes, err := ctx.Repo.GitRepo.GetCommit("refs/notes/commits")
+	if err == nil {
+		entry, err := notes.GetTreeEntryByPath(commitID)
+		if err == nil {
+			blob := entry.Blob()
+			dataRc, err := blob.DataAsync()
+			if err == nil {
+				d, err := ioutil.ReadAll(dataRc)
+				if err == nil {
```

Comment:
In case there is error dataRc won't be closed

Reply:
Fixed, thanks.